manifests/custom_config.pp

```diff
@@ -49,18 +49,18 @@
 #   show_diff property for configuration file resource
 #
 define apache::custom_config (
-  Enum['absent', 'present'] $ensure = 'present',
-  $confdir                          = $apache::confd_dir,
-  $content                          = undef,
-  $priority                         = '25',
-  $source                           = undef,
-  $verify_command                   = $apache::params::verify_command,
-  Boolean $verify_config            = true,
-  $filename                         = undef,
-  $owner                            = undef,
-  $group                            = undef,
-  $file_mode                        = undef,
-  Boolean $show_diff                = true,
+  Enum['absent', 'present'] $ensure         = 'present',
+  String $confdir                           = $apache::confd_dir,
+  Optional[String] $content                 = undef,
+  Variant[Integer,String,Boolean] $priority = '25',
```

Comment:
In what case could priority be a boolean?

Reply:
It can be set as boolean false to disable it and set the input to an empty string.

Reply:
That kind of option seems to be pretty common in the code